gsa/src/web/pages/cves/filterdialog.js

```diff
@@ -25,28 +25,8 @@ const SORT_FIELDS = [
     displayName: _l('Name'),
   },
   {
-    name: 'vector',
-    displayName: _l('Vector'),
-  },
-  {
-    name: 'complexity',
-    displayName: _l('Complexity'),
-  },
-  {
-    name: 'authentication',
-    displayName: _l('Authentication'),
-  },
-  {
-    name: 'confidentiality_impact',
-    displayName: _l('Confidentiality Impact'),
-  },
-  {
-    name: 'integrity_impact',
-    displayName: _l('Integrity Impact'),
-  },
-  {
-    name: 'availability_impact',
-    displayName: _l('Availability Impact'),
```

Comment:
Why has this code been removed?

Reply:
We refactored the CVE table https://github.com/greenbone/gsa/pull/2583 here.
We do not list these values in the table anymore and I missed removing that filter.